pkg/providers/instancetype/types.go

```diff
@@ -139,13 +139,20 @@ func computeRequirements(info *ec2.InstanceTypeInfo, offerings cloudprovider.Off
 	if family, ok := amiFamily.(*amifamily.Windows); ok {
 		requirements.Get(v1.LabelWindowsBuild).Insert(family.Build)
 	}
-	// Trn1 Accelerators
-	// TODO: remove function once DescribeInstanceTypes contains the accelerator data
-	// Values found from: https://aws.amazon.com/ec2/instance-types/trn1/
-	if strings.HasPrefix(*info.InstanceType, "trn1") {
-		requirements.Get(v1beta1.LabelInstanceAcceleratorName).Insert(lowerKabobCase("Inferentia"))
+	// AWS Neuron Accelerators
+	if info.NeuronInfo != nil && len(info.NeuronInfo.NeuronDevices) == 1 {
```

Comment:
Quick sanity check: We always expect this slice of info to have length 1? Is it possible that it could have some length other than 1 ever?